server/models/bill.coffee

```diff
@@ -13,6 +13,10 @@ module.exports = Bill = cozydb.getModel 'Bill',
     fileId: String
     content: String
     isRefund: Boolean
+    clientId: String # Client number in vendor CRM
+    number: String # Bill Id, in vendor CRM
```

Comment:
Maybe rename it as `vendorBillId` (seems clearer to me about what it should contain, `number` could be understood as `amount`). Also, what is it useful for?